GSA_Adapter/Convert/AdapterID.cs

```diff
@@ -21,9 +21,9 @@
  */
 
 
-namespace BH.Engine.GSA
+namespace BH.Adapter.GSA
 {
-    public partial class Convert
+    public static partial class Convert
     {
         public const string AdapterIdName = "GSA_id";
```

Comment:
My old idea of moving this field into the Adapter itself now should work fine, no?

I'd much prefer to see this added as a property of the Adapter.

Then we could delete this file.

Reply:
Do you mean putting this as a static property of the adapter? Could do that, even if it feels a bit confusing to have a static and a non-static property on the adapter returning the same thing?

Reply:
> Do you mean putting this as a static property of the adapter? Could do that, even if it feels a bit confusing to have a static and a non-static property on the adapter returning the same thing?

[Isn't that what we do anyway?](https://github.com/BHoM/GSA_Toolkit/blob/c5bcd1af40a51a9e1e850202523e5f00621edd1b/GSA_Adapter/GSAAdapter.cs#L46) 😄 
If it feels confusing it's because it's confusing, however, maybe putting the two "closer together", perhaps adding a description to the static property, it could help?

Reply:
difference there is that one sits on the adapter and one outside, but I get your point and agree..

Reply:
Yes one "sits outside" because it formerly _had_ to be this way, but now that feels unnecessary and unjustified, I think

Reply:
yep, get your point. Will move it in and see how it feels. Just feeling a bit unease having two of basically the same property, one static and one normal.